cmd/oomhero/main.go

```diff
@@ -10,53 +10,23 @@ import (
 )
 
 var (
-	warning  uint64 = 75
-	critical uint64 = 90
-	cooldown uint64 = 1
+	warning  uint64        = 75
+	critical uint64        = 90
+	cooldown time.Duration = 1 * time.Second
```

Comment:
To keep this short this could be set to something like `time.Duration = time.Second`, no need to multiply it by 1.

Reply:
Changed to `cooldown time.Duration = time.Second`